src/main/resources/db/changelog/ddl/0010_create_temporary_certificate_tables.xml

```diff
@@ -0,0 +1,121 @@
+                             references="electoral_registration_office(id)"/>
+            </column>
+            <column name="issuing_authority_cy" type="varchar(255)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="welsh_ero_id" type="uuid">
+                <constraints nullable="true"
+                             foreignKeyName="fk_temporary_certificate_welsh_ero_id"
+                             references="electoral_registration_office(id)"/>
+            </column>
+
+            <column name="first_name" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="middle_names" type="varchar(255)"/>
+            <column name="surname" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="certificate_language" type="varchar(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="photo_location_arn" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="issue_date" type="date">
+                <constraints nullable="false"/>
+            </column>
+            <column name="valid_on_date" type="date">
+                <constraints nullable="false"/>
+            </column>
+            <column name="status" type="varchar(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="user_id" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="date_created" type="timestamp">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created_by" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="version" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="nathan.russell@valtech.com" id="0010-EIP1-3610-create-temporary-certificate-status" context="ddl">
+        <createTable tableName="temporary_certificate_status">
+            <column name="id" type="uuid">
+                <constraints nullable="false" primaryKey="true" unique="true"
+                             primaryKeyName="temporary_certificate_status_pkey"/>
+            </column>
+            <column name="temporary_certificate_id" type="uuid">
+                <constraints nullable="false"
+                             foreignKeyName="fk_temporary_certificate_status_temporary_certificate"
+                             references="temporary_certificate(id)"/>
+            </column>
+            <column name="status" type="varchar(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="event_date_time" type="timestamp">
```

Comment:
Do you think we need `event_date_time` for this table? 

As far as I can remember, we are getting this value from the print-provider for the actual certificates (which will be different from the `date_created` values), but for the temporary certificate I guess it will be the nearly identical to the `date_created` value (I assume there will be an endpoint, if required, to update the status and the `event_date_time` will be the instant that button is pressed and it will update the status of the temporary certificate synchronously).

Reply:
Good point; yes; I'll get rid of `event_date_time` 👍